openstack/clustering/v1/profiletypes/testing/fixtures.go

```diff
@@ -0,0 +1,244 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/clustering/v1/profiletypes"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	fake "github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+const ProfileTypeRequestID = "req-7328d1b0-9945-456f-b2cd-5166b77d14a8"
+const ListResponse = `
+{
+	"profile_types": [
+		{
+			"name": "os.nova.server-1.0",
+			"schema": {
+				"context": {
+					"description": "Customized security context for operating containers.",
+					"required": false,
+					"type": "Map",
+					"updatable": false
+				},
+				"name": {
+					"description": "The name of the container.",
+					"required": false,
+					"type": "Map",
+					"updatable": false
+				}
+			}
+		},
+		{
+			"name": "os.heat.stack-1.0",
+			"schema": {
+				"context": {
+					"default": {},
+					"description": "A dictionary for specifying the customized context for stack operations",
+					"required": false,
+					"type": "Map",
+					"updatable": false
+				},
+				"disable_rollback": {
+					"default": true,
+					"description": "A boolean specifying whether a stack operation can be rolled back.",
+					"required": false,
+					"type": "Boolean",
+					"updatable": true
+				},
+				"environment": {
+					"default": {},
+					"description": "A map that specifies the environment used for stack operations.",
+					"required": false,
+					"type": "Map",
+					"updatable": true
+				},
+				"files": {
+					"default": {},
+					"description": "Contents of files referenced by the template, if any.",
+					"required": false,
+					"type": "Map",
+					"updatable": true
+				}
+			},
+			"support_status": {
+				"1.0": [
+					{
+						"status": "SUPPORTED",
+						 "since": "2016.04"
+					}
+				]
+			}
+		}
+	]
+}
+`
+
+const GetResponse1 = `
+{
+	"profile_type": {
+		"name": "os.nova.server-1.0",
+		"schema": {
+			"context": {
+				"description": "Customized security context for operating containers.",
+				"required": false,
+				"type": "Map",
+				"updatable": false
+			},
+			"name": {
+				"description": "The name of the container.",
+				"required": false,
+				"type": "Map",
+				"updatable": false
+			}
+		}
+	}
+}
+`
+
+const GetResponse15 = `
```

Comment:
I might be overlooking something obvious but I don't see how this microversion response is different from the standard response:

https://github.com/openstack/senlin/blob/e5bc16b3fea2f10dead365f983c5a937f23d72b0/senlin/api/openstack/v1/profile_types.py#L35-L38

Reply:
Perfect. I must have been comparing the wrong fixtures. Thank you :)